book/src/late-block-re-orgs.md

```diff
@@ -0,0 +1,59 @@
+# Late Block Re-orgs
+
+Since v3.3.0 Lighthouse will opportunistically re-org late blocks when proposing.
+
+This feature is intended to disincentivise late blocks and improve network health. Proposing a
+re-orging block is also more profitable for the proposer because it increases the number of
+attestations and transactions that can be included.
+
+## Command line flags
+
+There are three flags which control the re-orging behaviour:
+
+* `--disable-proposer-reorgs`: turn re-orging off (it's on by default).
+* `--proposer-reorg-threshold N`: attempt to orphan blocks with less than N% of the committee vote. If this parameter isn't set then N defaults to 20% when the feature is enabled.
+* `--proposer-reorg-participation-threshold N`: only attempt to re-org late blocks when the on-chain participation is approximately N% or greater. Default is 70%.
```

Comment:
A more conservative value like 95% might be more appropriate to start with. I chose 70% because it implies finality most of the tim) and is viable on testnets like Goerli where participation hovers around ~80%.

We could put in a 90/95% default for mainnet and 70% for everything else?

Reply:
> We could put in a 90/95% default for mainnet and 70% for everything else?

I'm down with this! I'd be tempted to go 95% for mainnet, just for the sake of caution. I don't have a strong argument to back this up, though. I think low values on testnets are a good idea to test the feature.